docs/about/technical_steering_committee.md

```diff
@@ -36,18 +40,42 @@ On this page we describe:
 ## Requirements to become a maintainer
 
 Just contributing does not make you a maintainer; you need to demonstrate commitment to Kedro's long-term success by
-working with existing maintainers for a period of time.
+working with existing maintainers for at least one year.
+
+On top of the [committer responsibilities](#responsibilities-of-a-committer), we look for people who can do at least some of the following:
+
+- Steer the direction of the project to ensure it retains relevance, including by bringing awareness of industry trends to the team and connecting the project to complementary initiatives
+- Give talks about the project at conferences and online events, write blog posts, or other forms of advocacy
+- Show excitement about the future of Kedro
+- In case of contributors from the community who haven't been previously appointed as committers,
+  show evidence of already having started pull requests and code reviews under the guidance of maintainers;
+  including asking for help where needed
```

Comment:
```suggestion
```
Is there any real reason for this carve-out? Contributor -> committer -> maintainer seems standard enough.